### ISSUE TITLE
Application not working in production mode

### ISSUE THREAD

**Issue by themastersoda** [Urigo/Ionic2CLI-Meteor-WhatsApp#34]
Hey, thanks for the tutorial! Everything's been working just fine while developing my application, however, when i decided to build it in production mode (for AOT compiling and other stuff), it always fails with error while building:

```
Error: Failed to find Angular bootstrapping code in C:/xampp/htdocs/PROJECT/src/app/main.ts. Please update C:/xampp/htdocs/PROJECT/src/app/main.ts to match the following: https://github.com/driftyco/ionic2-app-base/blob/master/src/app/main.ts
```

Here's how my main.ts looks:

```
import 'meteor-client-side';
import 'accounts-base-client-side';
import 'accounts-password-client-side';
import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
import { AppModule } from './app.module';

Meteor.startup(() => {
  platformBrowserDynamic().bootstrapModule(AppModule);
  Tracker.autorun(() => {
    if (Meteor.loggingIn()) return;
  });
});
```

Don't mind the tracker.autorun part - i'm using older version of angular2-meteor, but that is not related to the issue.

I'm trying to build for production using `ionic build android --prod`

**Comment by gotu** [comment]
Apparently this is an Ionic issue: https://github.com/driftyco/ionic-app-scripts/issues/594

**Comment by gotu** [comment]
This is fixed in ionic/app-scripts 1.1.0 https://github.com/driftyco/ionic-app-scripts/releases/tag/v1.1.0

**Comment by themastersoda** [comment]
Yes, it is finally working. Thanks for notifying :)
